### ISSUE TITLE
Order discussion threads by "In favour" and "Against"

### ISSUE THREAD

**Issue by xabier** [AjuntamentdeBarcelona/decidim.barcelona#705]
In order to improve deliberation and reasoning within discussion threads the "in favour" or "against" labels could be used to order the threads. First, easiest, implementation should order them as first most voted comments "In favour" down to the least voted for and then "Against" comments also in descending order. Alternatively, slightly more complicated, left column "In favour" and right "Against" (or viceversa), in descending order, and, if responsiveness forces single column, the orderding should alternate most "In favour" with most voted "Against".

**Comment by josepjaume** [comment]
Hi @xabier, thanks for the idea, it's definitely an improvement. You probably marked it as `easy` because it'd be a nice idea for the hackathon, but I'd concentrate all efforts in `decidim-next`. 

We've opened a `planning` project where we discuss all new (and re-implementations of old) features. It's a good opportunity to describe (and possible re-imagine) the comments feature:

https://github.com/codegram/decidim/projects/4

I've opened an issue for comments in the `ideas` column: https://github.com/codegram/decidim/issues/160 . We can use that to discuss what we'd want for a "comments" feature.
